Dockerfile

```diff
@@ -14,6 +14,9 @@ RUN pip install --upgrade /opt/appengine-compat /opt/appengine-vmruntime
 COPY resources/requirements.txt /opt/requirements.txt
 RUN pip install --upgrade -r /opt/requirements.txt
 
+# Install the Cloud Debugger
```

Comment:
Just add this to `resources/requirements.txt`. There's no need for this to have its own entry in the Dockerfile.